pyproject.toml

```diff
@@ -188,3 +189,8 @@ exclude_also = [
     "@(typing(_extensions)?\\.)?overload",
     "if (typing(_extensions)?\\.)?TYPE_CHECKING:"
 ]
+
+[tool.vulture]
+min_confidence = 80
+paths = ["airflow/"]
+sort_by_size = true
```

Comment:
I didn't expect to see two changes to this file, what does this one do?

Reply:
Oops, got code bleed from something else I was working on. Thanks for catching!

Reply:
Removed now.